tools/devctr/Dockerfile.x86_64

```diff
@@ -75,7 +75,9 @@ RUN apt-get update \
         setuptools \
         wheel \
     && python3 -m pip install --upgrade pip \ 
-    && gem install chef-utils:16.6.14 mdl
+    && gem install \
+        chef-utils:16.6.14 \
+        mdl:0.11.0
```

Comment:
This will also need to be set in the aarch64 Dockerfile

Reply:
arm does not run mdl

Reply:
It doesn't seem that `Dockerfile.aarch64` uses these dependencies.

Reply:
doh! LGTM then